framework/src/araui/libs/vec.c

```diff
@@ -0,0 +1,113 @@
+/** 
```

Comment:
You should add the copyright statement.
Please, check this through the whole files.

Reply:
Fixed